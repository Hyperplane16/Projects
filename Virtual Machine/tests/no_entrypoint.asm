FUNC LABEL 2
    CAL VAL 1
    ADD REG 0 REG 1
    PRINT REG 0
    RET
FUNC LABEL 1
    MOV REG 1 VAL 10
    RET
