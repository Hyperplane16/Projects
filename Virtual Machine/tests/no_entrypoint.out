0 functions of label 0 found.
1